README.md

```diff
@@ -15,9 +13,15 @@ Including this repo within your own will:
 
 ## Setup in code repo
 
-1. Add this repo as a submodule into `./scripts/` (if the directory doesn't exist, it will need to be created). To include the submodule run `git submodule add https://github.com/nhsuk/ci-deployment.git scripts/ci-deployment`
+1. Create an executable `scripts/deploy.sh` file with the following contents:
+```
+#!/bin/sh
+
+git clone https://github.com/nhsuk/ci-deployment.git scripts/ci-deployment
+bash ./scripts/ci-deployment/deploy.sh
+```
 2. Add a `docker-compose.yml` and optionally, a `rancher-compose.yml` file to `./rancher-config/`, located in the root of the project
-3. **(Travis only)** Update the target repo's `.travis.yml` to include the sections included within the example `.travis.yml` in this repo
+3. **(Travis only)** Update the target repo's `.travis.yml` to include the sections included within the example `.travis.yml` in this repo (see [travis/.travis-ci.yaml.example](https://github.com/nhsuk/ci-deployment/blob/master/travis/.travis.yml.example))
```

Comment:
Sorry, I should have mentioned this first time...the file can be linked to just by `[travis/.travis.yaml.example](travis/.travis.yml.example)` - this will work regardless of which branch the readme is on. And the file is named `travis/.travis.yml.example`.